lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -399,6 +401,20 @@ instance Buildable TxStatus where
         InLedger -> "in ledger"
         Invalidated -> "invalidated"
 
+instance FromText TxStatus where
+    fromText txt = case txt of
+        "pending" -> Right Pending
+        "in_ledger" -> Right InLedger
+        "invalidated" -> Right Invalidated
+        _ ->
+            Left . TextDecodingError $ show txt
+                   <> " is neither \"pending\", \"in_ledger\", nor \"invalidated\""
```

Comment:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Suggest:
```hs
            Left . TextDecodingError $ show txt
                <> " is neither \"pending\", \"in_ledger\", nor \"invalidated\""
```